src/openrct2/object/RideObject.cpp

```diff
@@ -857,13 +857,13 @@ rct_ride_entry_vehicle RideObject::ReadJsonCar(json_t& jCar)
             { "VEHICLE_ENTRY_FLAG_5", VEHICLE_ENTRY_FLAG_REVERSER_PASSENGER_CAR },
             { "VEHICLE_ENTRY_FLAG_HAS_INVERTED_SPRITE_SET", VEHICLE_ENTRY_FLAG_HAS_INVERTED_SPRITE_SET },
             { "VEHICLE_ENTRY_FLAG_DODGEM_INUSE_LIGHTS", VEHICLE_ENTRY_FLAG_DODGEM_INUSE_LIGHTS },
-            { "VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_2", VEHICLE_ENTRY_FLAG_ENABLE_ADDITIONAL_COLOUR_2 },
+            { "VEHICLE_ENTRY_FLAG_ENABLE_TERNARY_COLOUR", VEHICLE_ENTRY_FLAG_ENABLE_TERNARY_COLOUR },
```

Comment:
You'll need to keep the left hand side of this the old name as there will be object files in existence that use the old name. You can see L827 this has happened before. It's a bit annoying but we have to do this so that community made objects continue to work correctly.

Reply:
Understandable! Fixed.